packages/asana/test/integration.js

```diff
@@ -0,0 +1,172 @@
+import { expect } from 'chai';
+import configuration from '../tmp/creds.json' assert { type: 'json' };
+import {
+  getTask,
+  getTasks,
+  createTask,
+  updateTask,
+  upsertTask,
+  createTaskStory,
+  request,
+  execute,
+} from '../dist/index.js';
+
+const state = { configuration };
+
+describe('Integration tests', () => {
+  let createdTaskGid;
+  const testProjectGid = configuration.projectId; //'1206930238111330'; // Replace with a valid project GID from your Asana workspace
```

Comment:
What steps does one need to do to get these tests to pass?

And why would anyone want to use a different project id? These are our unit tests. If anyone wants to run the integration tests we should give them our credentials